Tests/NIOExtrasTests/FixedLengthFrameDecoderTest.swift

```diff
@@ -115,30 +116,22 @@ class FixedLengthFrameDecoderTest: XCTestCase {
         XCTAssertTrue(try channel.finish().isClean)
     }
 
-    func testCloseInChannelRead() {
```

Comment:
just in case anyone is surprised: this test was in the wrong test suite, I moved it.